CHANGES.rst

```diff
@@ -2,6 +2,18 @@
 Changelog
 =========
 
+Version 4.0.0 (2020-07-XX)
```

Comment:
Are we breaking stuff?

Reply:
Since this is not a breaking release, I'd like to hold off with the 4.0 unless we want to do a short sprint to remove at least the currently deprecated stuff

Reply:
updated the version as 3.11.0